package.json

```diff
@@ -147,6 +146,12 @@
     "configuration": {
       "title": "Quarkus Tools",
       "properties": {
+        "redhat.telemetry.enabled": {
```

Comment:
According to https://github.com/redhat-developer/vscode-redhat-telemetry#contribute-the-redhattelemetryenabled-preference , `Unless your extension already depends on a telemetry-enabled Red Hat extension, it needs to declare the redhat.telemetry.enabled preference in its package.json, like:` . So does this need to be here ? This depends on vscode-microprofile, which defines this property.